recipe/meta.yaml

```diff
@@ -36,8 +34,10 @@ requirements:
     - {{ posix }}zip               # [win]
     - cross-r-base {{ r_base }}    # [build_platform != target_platform]
     - llvm-openmp                  # [osx]
-    - libgomp                      # [linux]
+    - libgomp                      # [not osx]
   host:
+    - llvm-openmp                  # [osx]
+    - libgomp                      # [not osx]
```

Comment:
Would think changing the selector in `requirements/build` would have been sufficient here. Is there a reason it was not?

Reply:
It turns out there is actually another, separate issue with `llvm-openmp` on macOS needing to be in both `host` and build as explained here by @mfansler: https://github.com/conda-forge/r-rdimtools-feedstock/pull/2#issuecomment-2438103377.  Essentially it seems that `conda-build` can end up pinning two different versions of `llvm-openmp` and this creates some unsolvable dependency conflicts.  That particular behavior sounds like a possible bug, which may or may not be related to the issues we're having with `run_exports` for `libgomp`.